apps/voting/app/src/script.js

```diff
@@ -169,12 +219,39 @@ async function startVote(state, { creator, metadata, voteId }) {
   }))
 }
 
+async function getAccountVotes({ connectedAccount = '', votes = [] }) {
```

Comment:
Probably don't need the default args for both of these functions, I think web3 would complain if we tried to use an empty address too.